custom_components/renault/manifest.json

```
@@ -5,8 +5,9 @@
   "documentation": "https://www.home-assistant.io/integrations/renault",
   "issue_tracker": "https://github.com/epenet/hassRenaultZE/issues",
   "requirements": [
-    "pyze==0.6.0",
-    "renault-api==0.0.1a0"
+    "renault-api==0.0.1.dev1606315769"
```

To facilitate the test of the branch you can use a github version of the Python library.
```suggestion
    "git+https://github.com/hacf-fr/renault-api.git@dev#develop==0.0.1.dev1606315769"
```
It will use current dev branch on GitHub. Then you can do a pre-release and let people use beta release in HACS or easily install it by copying file.

---

Thanks @oncleben31 I have updated the manifest file.